### ISSUE TITLE
Installation fails on El Capitan

### ISSUE THREAD

**Issue by stringbeans** [monfresh/laptop#44]
Hi there,

When I use the command `curl -s https://raw.githubusercontent.com/monfresh/laptop/master/laptop | sh -` it fails because it prompts about zsh and doesnt allow the user to answer

```
Johns-MBP:api-v2-docs john$ curl -s https://raw.githubusercontent.com/monfresh/laptop/master/laptop | sh -
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  8864  100  8864    0     0  40619      0 --:--:-- --:--:-- --:--:-- 40847
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   199  100   199    0     0    964      0 --:--:-- --:--:-- --:--:--   970
Want to switch your shell from the default bash to zsh?
Both work fine for development, and zsh has some extra 
features for customization and tab completion.
If you aren't sure or don't care, we recommend zsh.
Press y to switch to zsh, n to keep bash: failed
```

**Comment by monfresh** [comment]
Interesting. Thanks for the report! I tested this on macOS Sierra, but not El Capitan. I will try it and see what's going on. In the meantime, if you feel comfortable trying to get around this yourself, follow these steps:

1. Open up the file called `mac` in your Home folder in a text editor, and replace all of these lines:
https://github.com/monfresh/laptop/blob/master/mac#L58-L97

with this line:
```bash
create_bash_profile_and_set_it_as_shell_file
```
2. Save the file
3. Go back to Terminal
4. Run `bash mac 2>&1 | tee ~/laptop.log`

Let me know if that worked.

**Comment by monfresh** [comment]
@stringbeans I think I found the solution. Can you try this please?

``` 
bash <(curl -s https://raw.githubusercontent.com/monfresh/laptop/master/laptop)
```

**Comment by stringbeans** [comment]
worked!! :)

**Comment by monfresh** [comment]
Nice. I'll push the fix now. Thanks!

**Comment by monfresh** [comment]
Fixed with #45
